GameDefinition/Content/ItemKindActor.hs

```diff
@@ -357,7 +356,7 @@ komodoDragon = ItemKind  -- bad hearing; regeneration makes it very powerful
   , iaspects = [ AddMaxHP 41, AddMaxCalm 60, AddSpeed 18, AddNocto 2 ]
   , ieffects = []
   , ifeature = [Durable, Identified]
-  , idesc    = ""
+  , idesc    = "Larger and more aggressive than any other lizard."
```

Comment:
Oh yes. And you don't spoil the 'dragon' fantasy pun, by over-explaining it's an actual real species. :)